In [1]:
import os
import sys
import nglview as nv
import MDAnalysis as mda

In [2]:
Topology="../Trajectories_and_tprs/No_solvent_CAs.tpr"
Trajectory="../Trajectories_and_tprs/No_solvent_CAs.xtc"

In [3]:
u=mda.Universe(Topology,Trajectory)

/home/dhanushkaw/anaconda3/envs/Analysis/lib/python3.9/site-packages/MDAnalysis/coordinates/XDR.py:237: UserWarning: Reload offsets from trajectory
 ctime or size or n_atoms did not match
  warnings.warn("Reload offsets from trajectory\n "


In [4]:
sel=u.select_atoms('resname RAMP PMB')

In [5]:
w=nv.show_mdanalysis(sel)
w

NGLWidget(max_frame=10000)

In [8]:
PMB1s=u.select_atoms("resname PMB")

In [9]:
print(len(PMB1s))
print(PMB1s)


3760
<AtomGroup [<Atom 176135: C1 of type HT of resname PMB, resid 1457 and segid seg_0_Something clever....>, <Atom 176136: C2 of type OT of resname PMB, resid 1458 and segid seg_0_Something clever....>, <Atom 176137: C3 of type HT of resname PMB, resid 1459 and segid seg_0_Something clever....>, ..., <Atom 179892: H1Y of type OT of resname PMB, resid 5214 and segid seg_0_Something clever....>, <Atom 179893: H1Z of type HT of resname PMB, resid 5215 and segid seg_0_Something clever....>, <Atom 179894: H1W of type HT of resname PMB, resid 5216 and segid seg_0_Something clever....>]>


- resid 1457:1644 - 1:188
- resid 1645:1832 - 189:376
- resid 1833:2020 - 377:564 - not
- resid 2021:2208 - 565:752
- resid 2209:2396 - 753:940 -not
- resid 2397:2584 - 941:1128
- resid 2585:2772 - 1129:1316
- resid 2773:2960 - 1317:1504
- resid 2961:3148 - 1505:1692 - not
- resid 3149:3336 - 1693:1880
- resid 3337:3524 - 1881:2068
- resid 3525:3712 - 2069:2256 - not
- resid 3713:3900 - 2257:2444 - not
- resid 3901:4088 - 2445:2632 - not
- resid 4089:4276 - 2633:2820
- resid 4277:4464 - 2821:3008 - LPS-bound
- resid 4465:4652 - 3009:3196 - not
- resid 4653:4840 - 3197:3384
- resid 4841:5028 - 3385:3572
- resid 5029:5216 - 3573:3760


